### DISCUSSION TITLE
Orbit and Backup

### DISCUSSION THREAD

**Discussion by happytodev** [ryangjchandler/orbit#145] [Q&A] [answered]
Hi Ryan,

what is, from your thoughts, the best way to backup data from Orbit ?

Do we need to just backup content folder ?

Or do we need to backup sqlite database too?

Thank you for your answer

**Comment by ryangjchandler** [comment, marked as answer]
@happytodev Backing up the content folder should be enough - the SQLite database shouldn't be needed since the next time you try to use one of the Orbit models it will be created anyway.

The content is the main source of data truth.

**Reply by happytodev** [reply]
Ok thanks for this clear answer.

As maybe you know I'm developing a CMS based on Orbit and Filament. So, during development, when I add fields to a previous model, I have errors that this field doesn't exist.

Do you think there is a good way to avoid this?
What do you recommend as best practice?

**Reply by ryangjchandler** [reply]
@happytodev You need to make sure the field is nullable and clear the cache using the provided artisan command, or update the existing content to have the new field.

**Reply by happytodev** [reply]
Yes I see, thank you for the tips
